src/activeTodos/model.js

```diff
@@ -0,0 +1,11 @@
+const {DataTypes} = require("sequelize")
+const connection = require("../db/connection")
+
+const ActiveTodo = connection.define("ActiveTodo", {
+    completedTasks: {
```

Comment:
The design specs say that this field should be called "todo", rather than "completedTasks" (addActiveTodo.txt).

Be careful when copying code. Double-check your files.

Reply:
I missed that! Thanks for pointing it out! Just changed it now